README.md

```diff
@@ -1053,6 +1053,10 @@ This repository is being kept up to date thanks to the help from the community.
 🌻 [Gabe Kuslansky](https://github.com/GabeKuslansky),
 🌻 [Ron Gross](https://github.com/ripper234),
 🌻 [Valeri Karpov](https://github.com/vkarpov15)
+🌻 [Sergio](https://github.com/imsergiobernal),
+🌻 [Duarte Mendes](https://github.com/duartemendes),
+🌻 [Nikola Telkedzhiev](https://github.com/ntelkedzhiev),
+🌻 [Vitor Godoy](https://github.com/vitordagamagodoy)
 
```

Comment:
Seems like we missed awarding a flower to @manishsaraan for contributing https://github.com/i0natan/nodebestpractices/pull/241